nlp-model.py

```diff
@@ -0,0 +1,52 @@
+import re
+from nltk.stem import WordNetLemmatizer
+nltk.download('wordnet')
+lemmatizer = WordNetLemmatizer()
+
+
+class PreprocessText(object):
+    """
+    Sentiment relevant text properties. 
+    """
+    
+    def __init__(self, text):
+        
+        self.text = text
+        self.clean_text = self._clean_text()
```

Comment:
This should be removed in favor of just renaming the `_clean_text` method to `clean_text`.
```suggestion
```